sienna_database_gen/pydantic_schemas.py

```diff
@@ -10,16 +10,60 @@
 from enum import StrEnum
 from pydantic import (
     BaseModel,
-    Field,
     model_validator,
 )
 from pydantic.json_schema import GenerateJsonSchema
 
-from typing import Any, ClassVar, Annotated, get_type_hints
+from typing import ClassVar, Annotated, get_type_hints
 from sqlalchemy import Double, ForeignKey, Table, Column, Integer, Text
 
-from R2X_schemas.enums import ACBusTypes, PrimeMoversType, ThermalFuels
-from R2X_schemas.units import ApparentPower, Voltage
+
+class PrimeMoversType(StrEnum):
+    """EIA prime mover codes."""
+
+    BA = "BA"
+    BT = "BT"
+    CA = "CA"
+    CC = "CC"
+    CE = "CE"
+    CP = "CP"
+    CS = "CSV"
+    CT = "CT"
+    ES = "ES"
+    FC = "FC"
+    FW = "FW"
+    GT = "GT"
+    HA = "HA"
+    HB = "HB"
+    HK = "HK"
+    HY = "HY"
+    IC = "IC"
+    PS = "PS"
+    OT = "OT"
+    ST = "ST"
+    PV = "PV"
```

Comment:
```suggestion
    PVe = "PVe"
```

Reply:
How do you feel about getting rid of these pydantic schemas right now? These are the ones I had as part of the initial prototype.

Reply:
go for it.

Reply:
I tried to use the powersystemcshemas for these enums. It seems to actually be nontrivial, and it defaults to using some sort of weird validation instead of a proper enum.

Reply:
Ok, I finally figured out how to do this with the help of Daniel Thom: `openapi-generator generate -i openapi.json --inline-schema-options RESOLVE_INLINE_ENUMS=true -g python -o sienna_schema`.